lib/bitreserve/operations/all.rb

```diff
@@ -0,0 +1,26 @@
+module Bitreserve
+  module Operations
+    module All
+      module ClassMethods
+        def all
+          data = Request.perform(:get, Kernel.const_get("Bitreserve::Endpoints::#{name.split('::').last.upcase}"))
+          parse_objects data
+        end
+
+        private
+
+        def parse_objects(data)
+          return [] unless data
+
+          data.map do |object_data|
+            new(object_data)
+          end
+        end
+      end
+
+      def self.included(base)
+        base.extend(ClassMethods)
+      end
```

Comment:
wouldn't this be more readable at the beginning of the module, instead of the end?